http_check/README.md

```diff
@@ -46,6 +46,7 @@ See the [sample http_check.d/conf.yaml][3] for a full list and description of av
 | `include_content`                | When set to `true`, the check includes the first 500 characters of the HTTP response body in notifications. The default value is `false`.                                                                                                        |
 | `collect_response_time`          | By default, the check collects the response time (in seconds) as the metric `network.http.response_time`. To disable, set this value to `false`.                                                                                                 |
 | `tls_verify`                     | Instructs the check to validate the TLS certificate of services when reaching to `url`.                                                                                                                                                          |
+| `tls_retrieve_non_validated_cert`| If `tls_verify` is set to `false`, setting this to `true` will allow the certificate to be examined for an expiration date.                                                                                                                      |
```

Comment:
```suggestion
| `tls_retrieve_non_validated_cert`| If `tls_verify` is set to `false`, setting this to `true` allows the certificate to be examined for an expiration date.                                                                                                                      |
```

Reply:
Minor suggestion per our [contributing guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense)

Reply:
updated...thanks for the link!